src/assignment/modals/AddCollectionModal.tsx

```diff
@@ -137,7 +137,7 @@ const AddCollectionModal: FunctionComponent<AddCollectionModalProps> = ({
 			const columnDataType: TableColumnDataType = (column?.dataType ||
 				TableColumnDataType.string) as TableColumnDataType;
 
-			let collections: Avo.Collection.Collection[];
+			let collections: Collection[];
 			if (activeView === AddCollectionTab.myCollections) {
 				collections = await CollectionService.fetchCollectionsByOwner(
```

Comment:
is this function name still acurate? Shouldn't this be something like:
fetchCollectionsByOwnerOrContributorProfileId?